synapse/lib/layer.py

```diff
@@ -1782,6 +1783,38 @@ async def truncate(self):
 
         await self._initLayerStorage()
 
+    # async def wipe(self, meta): ...
```

Comment:
Placeholder to wipe() using nodeedits *without* going through the view to fire triggers; I didn't see a need for it currently.